tests/test_datetime.py

```diff
@@ -25,7 +26,6 @@
         ),
         pytest.param('28/01/87', IsDatetime(approx=datetime(2000, 1, 1)), False, id='string-format-different'),
         pytest.param('foobar', IsDatetime(approx=datetime(2000, 1, 1)), False, id='string-format-wrong'),
-        pytest.param(datetime.now().isoformat(), IsNow(iso_string=True), True, id='isnow-str-true'),
```

Comment:
We already have equivalent test `test_is_now_str`, so I removed that test at all